.dotnet.azure/sdk/openai/Azure.AI.OpenAI/tests/VectorStoreTests.cs

```diff
@@ -70,7 +71,7 @@ public async Task CanCreateGetAndDeleteVectorStores()
             Assert.That(vectorStore.Status, Is.EqualTo(VectorStoreStatus.InProgress));
             Assert.That(vectorStore.Metadata?.TryGetValue("test-key", out string metadataValue) == true && metadataValue == "test-value");
         });
-        vectorStore = await client.GetVectorStoreAsync(vectorStore);
+        vectorStore = createVectorStoreOperation.GetVectorStore();
```

Comment:
Since this LRO has a convenience-layer, you shouldn't need to call the service to get the latest strongly-typed VectorStore model instance, but instead can get the `operation.Value` property, which should have deserialized the type from the last update response and set this property in the implementation of the derived types `UpdateStatus` method.

Reply:
I didn't know I could do that!